### PR TITLE
Create phd-course-dynamics-complex-systems.md

### PR DESCRIPTION
**Pull request by claudiobonanno**

### CODE REVIEW

File: content/posts/phd-course-dynamics-complex-systems.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
++++ 
+title = "Ph.D. course 'Dynamics of complex systems'" 
+date = "2024-04-05 11:00:04" 
+categories = ["news", "events"] 
++++
+
+The Ph.D. School in Mathematics of the *University of Pisa* offers the course on the **'Dynamics of complex systems'** by Stefano Galatolo, Roberto Castorrini, Matteo Tanzi, and Bastien Fernandez. 
+The course is in presence but can also be followed remotely.
```
Comment (claudiobonanno) [inline content/posts/phd-course-dynamics-complex-systems.md:8]: Change into "The course is in person but..."

Comment (mseri) [reply content/posts/phd-course-dynamics-complex-systems.md:8]: Why not
```suggestion
The course will be offered in an hybrid format.
```

**Comment by claudiobonanno** [comment]
Mah, ho visto usare l’espressione “in person” e per mia insicurezza sull’inglese ho pensato che fosse più corretta di “in presence”.Tu sei certamente più in grado di me di scegliere la forma più bella.Claudio Il giorno 5 apr 2024, alle ore 14:26, Marcello Seri ***@***.***> ha scritto:﻿
@mseri commented on this pull request.



In content/posts/phd-course-dynamics-complex-systems.md:
> @@ -0,0 +1,14 @@
++++ 
+title = "Ph.D. course 'Dynamics of complex systems'" 
+date = "2024-04-05 11:00:04" 
+categories = ["news", "events"] 
++++
+
+The Ph.D. School in Mathematics of the *University of Pisa* offers the course on the **'Dynamics of complex systems'** by Stefano Galatolo, Roberto Castorrini, Matteo Tanzi, and Bastien Fernandez. 
+The course is in presence but can also be followed remotely.

Why not
⬇️ Suggested change
-The course is in presence but can also be followed remotely.
+The course will be offered in an hybrid format.

File: content/posts/phd-course-dynamics-complex-systems.md (hunk 1 above)
Comment (mseri) [reply content/posts/phd-course-dynamics-complex-systems.md:8]:
```suggestion
This course will be conducted in person, with the option for remote participation.
```

**Pull request merged**